src/commands/lambda/functions/uninstrument.ts

```diff
@@ -1,28 +1,34 @@
-import {CloudWatchLogs, Lambda} from 'aws-sdk'
+import { CloudWatchLogs, Lambda } from 'aws-sdk'
 import {
   API_KEY_ENV_VAR,
   API_KEY_SECRET_ARN_ENV_VAR,
   CAPTURE_LAMBDA_PAYLOAD_ENV_VAR,
   DD_LAMBDA_EXTENSION_LAYER_NAME,
+  DOTNET_RUNTIME,
+  DOTNET_TRACER_HOME_ENV_VAR,
+  ENABLE_PROFILING_ENV_VAR,
   ENVIRONMENT_ENV_VAR,
   EXTRA_TAGS_ENV_VAR,
   FLUSH_TO_LOG_ENV_VAR,
   HANDLER_LOCATION,
+  INTEGRATIONS_ENV_VAR,
   KMS_API_KEY_ENV_VAR,
   LAMBDA_HANDLER_ENV_VAR,
   LOG_LEVEL_ENV_VAR,
   MERGE_XRAY_TRACES_ENV_VAR,
+  PROFILER_ENV_VAR,
+  PROFILER_PATH_ENV_VAR,
   Runtime,
   RUNTIME_LAYER_LOOKUP,
   SERVICE_ENV_VAR,
   SITE_ENV_VAR,
   TRACE_ENABLED_ENV_VAR,
   VERSION_ENV_VAR,
 } from '../constants'
-import {FunctionConfiguration, LogGroupConfiguration, TagConfiguration} from '../interfaces'
-import {calculateLogGroupRemoveRequest} from '../loggroup'
-import {calculateTagRemoveRequest} from '../tags'
-import {getLambdaFunctionConfigs, getLambdaFunctionConfigsFromRegex, getLayers, isSupportedRuntime} from './commons'
+import { FunctionConfiguration, LogGroupConfiguration, TagConfiguration } from '../interfaces'
```

Comment:
Nit: What is adding all of these extra spaces between `{}`? Is this something you're doing manually or do you have some kind of automated tool that is doing it?

Reply:
Must be when I run `yarn format` or it is eslint.